tests/test_retrieve_docket.py

```diff
@@ -0,0 +1,21 @@
+"""
+Test the retrieve_docket.py file
+"""
+import json
+import requests
+
+from c20_server.retrieve_docket import RetrieveDocket
+
+def create_an_instance_of_object():
+    a_test = RetrieveDocket()
+    assert a_test is not None
+
+def test_good_docket():
+    a_test = RetrieveDocket()
+
+    response = requests.get("https://api.data.gov:443/regulations/v3/docket" +
```

Comment:
This test will fail if

* The data.gov site is down
* The machine running tests is not connected to the internet
* etc.

It needs to be re-written with [request-mock](https://requests-mock.readthedocs.io/en/latest/) . This will also allow us to remove the API key.